examples/src/main/java/io/grpc/examples/header/HeaderClientInterceptor.java

```diff
@@ -0,0 +1,40 @@
+package io.grpc.examples.header;
+
+import io.grpc.*;
+
+import java.util.logging.Logger;
+
+/**
+ * interceptor to handle client header
+ * @author zhaohaifeng
+ * @since 2015-03-16
```

Comment:
I don't think @since adds much value for examples. It is normally helpful in telling you when an API was added, which doesn't quite apply to the examples.